server/middlewares/stats.js

```diff
@@ -71,13 +106,40 @@ const trackTxSubmissions = mung.json((body, req, res) => {
         : 'otherTxSubmissions'
     const txSubmissionSuccess = body.Right ? 'successful' : 'unsuccessful'
 
-    incrCountersBy(`${txSubmissionType}:${txSubmissionSuccess}`, 1)
-
-    if (txSubmissionSuccess === 'successful') {
-      const {txBody} = req.body
-      const txOutAmount = parseTxBodyOutAmount(txBody)
-
-      incrCountersBy(`${txSubmissionType}:sentOut`, txOutAmount)
+    try {
+      if (txSubmissionSuccess === 'successful') {
+        const {txBody} = req.body
+        const decodedTxBody = decodeTxBody(txBody)
+
+        const txOutAmount = parseTxBodyOutAmount(decodedTxBody)
+        if (txOutAmount > 0) {
+          trackEvent({
+            category: 'Payments',
+            action: `${txSubmissionType}:sentOutSuccess`,
+            label: 'successful payment',
+            value: txOutAmount,
+          })
+        }
+
+        const txChangeAmount = parseTxBodyChangeAmount(decodedTxBody)
```

Comment:
we are not really interested in the change amount but in the total amount being sent

Reply:
So that would be sum of sentAmount and changeAmount?

Reply:
Do you want to track both sentAmount and totalAmount separately? Or should we track only totalAmount?

Reply:
yes, that's the sum of all outputs, I would suggest using array.map for that. So I would track total amount and sentOut amount separately, both provide meaningful information - the total amount may serve to determine how is adalite daily volume related to Cardano total daily volume and the sent out amount serves to tell how big transactions are we actually dealing with from the point of view of the user.